bokeh/tests/test_properties.py

```diff
@@ -317,14 +317,28 @@ class IsDelegate(HasProps):
 
         class IncludesDelegateWithPrefix(HasProps):
             z = Include(IsDelegate, use_prefix=True)
-            z_y = String("world") # override the Include
+            z_y = Int(57) # override the Include
```

Comment:
if we think of Include conceptually as a mixin, allowing this is a bit sketchy, but the rationalization is that we do truly, fully replace the property - it isn't sticking around in an actual base class anywhere.